lib/emulator_rubocop.rb

```diff
@@ -1,8 +1,31 @@
 # frozen_string_literal: true
 
-require_relative "emulator_rubocop/version"
+require_relative 'modules/version'
+require_relative 'modules/warning_shower'
+require_relative 'modules/warning_finder'
 
-module EmulatorRubocop
-  class Error < StandardError; end
-  # Your code goes here...
+class EmulatorRubocop
+  include Modules
+
+  def self.call(args)
+    new(args).call
+  end
+
+  def initialize(args)
+    @arr_of_files_path = check_file_path?(args[0]) || Dir['./app/*.rb'] + Dir['./web/*.rb']
```

Comment:
```suggestion
    @file_paths = check_file_path?(args[0]) || Dir['./app/*.rb'] + Dir['./web/*.rb']
```
or 
```suggestion
    @load_paths = check_file_path?(args[0]) || Dir['./app/*.rb'] + Dir['./web/*.rb']
```
Something like that